### PR TITLE
Bitpack internal IDs

### PR DESCRIPTION
**Pull request by benjaminwinger**
I was going to benchmark this, as it adds an extra step to the decompression, but I'm not sure it's actually being used. Are internalIDs actually stored anywhere other than in rels?

### CODE REVIEW

File: src/storage/store/column_chunk.cpp
Comment (ray6080) [inline src/storage/store/column_chunk.cpp:125]: Can we move the switch case into a separate function? This is becoming quite large now in the constructor.

**Review by ray6080 — APPROVED** [review]

**Comment by ray6080** [comment]
> I was going to benchmark this, as it adds an extra step to the decompression, but I'm not sure it's actually being used. Are internalIDs actually stored anywhere other than in rels?

You're right. Currently it's not materialized in node tables.

File: src/storage/store/column_chunk.cpp
Comment (benjaminwinger) [reply src/storage/store/column_chunk.cpp:125]: Sure. I'd already simplified it somewhat in #2102 ([here](https://github.com/kuzudb/kuzu/pull/2102/files#diff-b7ddec24858a04e0feec0e25f3a5dca30c0672beb96db4147b64e044a84dcad6R95-R144)), so I could move that change into this PR. Is that simplification enough, or would you prefer moving everything into a separate function?

**Comment by ray6080** [comment]
@benjaminwinger , can we hold this PR a bit longer after the rel node group one #2246 ?

**Comment by benjaminwinger** [comment]
Yes, I was thinking it would make sense to wait until that's merged so it can be tested.

**Comment by ray6080** [comment]
I think this can be closed now. @benjaminwinger

**Comment by benjaminwinger** [comment]
Yes, let's close in favour of #2579

**Pull request closed without merging**
